src/libraries/System.DirectoryServices.AccountManagement/tests/ComputerPrincipalTest.cs

```diff
@@ -1,83 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-using Xunit;
-
-namespace System.DirectoryServices.AccountManagement.Tests
-{
-    public class ComputerPrincipalTest : PrincipalTest
-    {
-        [Fact]
-        public void Ctor_Context()
-        {
-            var context = new PrincipalContext(ContextType.Machine);
-            var principal = new ComputerPrincipal(context);
-            Assert.Same(context, principal.Context);
-            Assert.Empty(principal.ServicePrincipalNames);
-        }
-
-        [Fact]
-        public void Ctor_NullContext_ThrowsArgumentException()
-        {
-            AssertExtensions.Throws<ArgumentException>(null, () => new ComputerPrincipal(null));
-            AssertExtensions.Throws<ArgumentException>(null, () => new ComputerPrincipal(null, "samAccountName", "password", enabled: true));
-        }
-
-        [Fact]
-        public void Ctor_NullSamAccountName_ThrowsArgumentException()
-        {
-            var context = new PrincipalContext(ContextType.Machine);
-            AssertExtensions.Throws<ArgumentException>(null, () => new ComputerPrincipal(context, null, "password", enabled: true));
-        }
-
-        [Fact]
-        public void Ctor_EmptySamAccountName_ThrowsArgumentNullException()
-        {
-            var context = new PrincipalContext(ContextType.Machine);
-            AssertExtensions.Throws<ArgumentNullException>("value", null, () => new ComputerPrincipal(context, string.Empty, "password", enabled: true));
-        }
-
-        [Fact]
-        public void Ctor_NullPassword_ThrowsArgumentException()
-        {
-            var context = new PrincipalContext(ContextType.Machine);
-            AssertExtensions.Throws<ArgumentException>(null, () => new ComputerPrincipal(context, "samAccountName", null, enabled: true));
-        }
-
-        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoNorServerCore), nameof(PlatformDetection.IsNotWindowsIoTCore))]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/34442", TestPlatforms.Windows, TargetFrameworkMonikers.Netcoreapp, TestRuntimes.Mono)]
```

Comment:
What about this test? this one doesn't seem to have been migrated over to AccountManagementTests right?

Reply:
That's included here if I'm correct:
https://github.com/dotnet/runtime/blob/0ea9425bcf527b615a07f6cf5fe9936b6070cc0a/src/libraries/System.DirectoryServices.AccountManagement/tests/AccountManagementTests.cs#L22-L28
You could argue that not the exact same constructor signature was used, but that does not matter for this specific check, which is about the context.